src/main/java/com/PhamKornbluhGroup/PublicStashTabBulkAPICaller.java

```diff
@@ -0,0 +1,66 @@
+package com.PhamKornbluhGroup;
+
+import java.io.FileWriter;
+import java.net.HttpURLConnection;
+import java.net.URL;
+import java.util.Arrays;
+
+public class PublicStashTabBulkAPICaller {
```

Comment:
Consider renaming to APIHelper if this class is strictly for API-related uses, or rename to GGGAPIHandler, or some similar name to more concisely express the class' purpose.

Reply:
In our 1-on-1 discussion, we agreed that GGGAPIHandler would be the the most appropriate name, based on current and future functionality of the class.

Updated. Thanks